.github/workflows/check_compatibility_careamics.yaml

```diff
@@ -0,0 +1,49 @@
+name: check compatibility careamics
+
+concurrency: careamics
+
+on:
+  push:
+    branches:
+      - main
+    paths:
+      - .github/workflows/check_compatibility_careamics.yaml
+      - scripts/check_compatibility_careamics.py
+  workflow_dispatch:
+  schedule:
+    - cron: 0 1 * * *  # update compatibility once a day
+
+jobs:
+  run:
+    runs-on: ubuntu-latest
+    environment: 'production'
+    steps:
+    - uses: actions/checkout@v4
+    - name: Setup careamics env
+      uses: mamba-org/setup-micromamba@v1
+      with:
+        generate-run-shell: false
+        cache-downloads: true
+        environment-name: careamics
+        condarc: |
+          channels:
+            - conda-forge
+            - pytorch
+        create-args: >-
+          pytorch
+          torchvision
+          cpuonly
+          python=3.10
+          numpy
+          pip
+    - run: pip install careamics
```

Comment:
We now have a `conda-forge` careamics package, so this step could be removed and `careamics` added to the previous step!